tests/local/test_server.py

```diff
@@ -24,6 +24,11 @@ def test_main_post(self, mocker, client, event, action):
         mock_func = mock.MagicMock(return_value=True)
         mocker.patch('pep8speaks.utils.match_webhook_secret', mock_func)
         mocker.patch('pep8speaks.handlers.' + action, mock_func)
-        client.post(url_for('main'),
-                    headers={"X-GitHub-Event": event})
+        client.post(
+            url_for('main'),
+            headers={"X-GitHub-Event": event},
+            # TODO: This test is not representative of the real JSON payloads sent by Github and should be updated at
+            # some point to have a sample payload fixture for each of the above types.
+            json={"a": "value", "b": 1},
```

Comment:
This `json` param was required, but missing. Ideally the value would look more like a real payload, but those are hundreds/thousands of lines long and I didn't have a good example to dump in here.